src/main/java/tickbot/storage/DataStorage.java

```diff
@@ -0,0 +1,113 @@
+package tickbot.storage;
+
+import java.io.File;
+import java.io.FileInputStream;
+import java.io.FileOutputStream;
+import java.io.IOException;
+import java.io.InputStreamReader;
+import java.io.OutputStreamWriter;
+import java.time.DateTimeException;
+import java.time.LocalDate;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Objects;
```

Comment:
Maybe leave a line in between imports from different packages. `java.io`, `java.time` and `java.util` all count as different packages. Same for lines 18-24.

Reply:
Unfortunately this grouping is what `checkstyle` requires. If I add empty lines here `checkstyle` would complain about "Extra separation in import group".